agent/acl.go

```diff
@@ -104,6 +104,12 @@ func newACLManager(config *Config) (*aclManager, error) {
 					Policy: acl.PolicyWrite,
 				},
 			},
+			Nodes: []*acl.NodePolicy{
+				&acl.NodePolicy{
+					Name:   "",
```

Comment:
This is actually the prefix rule for which nodes this is allowed to read, not the name of the rule, so the empty prefix `""` means that it can see any node.